### DISCUSSION TITLE
I actually found a work around for my project. I added `.dropdown-content{ overflow-y: auto }` to my css and it scrolls properly for me.

### DISCUSSION THREAD

**Discussion by JoseSNubimetrics** [dowjones/react-dropdown-tree-select#600] [Show and tell]
I actually found a work around for my project. I added `.dropdown-content{ overflow-y: auto }` to my css and it scrolls properly for me.

_Originally posted by @lyleaigbedion in https://github.com/dowjones/react-dropdown-tree-select/issues/207#issuecomment-773656211_

It worked for you?? I tried to implement it but it didn´t work!

**Comment by sshettyalgo** [comment]
Did you find a work around or a solution for the problem? the other methods/solutions aren't working for me
